src/components/pages/Offers/Offer/Stocks/StockItem/StockItem.jsx

```diff
@@ -145,7 +145,7 @@ const StockItem = ({
             <DateInput
               ariaLabel="Date de l’événement"
               dateTime={beginningDate}
-              disabled={isOfferSynchronized || !isStockEditable}
+              disabled={isDisabled || isOfferSynchronized || !isStockEditable}
```

Comment:
```suggestion
              disabled={isOfferDisabled || isOfferSynchronized || !isStockEditable}
```
C'est une proposition, parce que du coup entre autre isStockEditable et isDisabled on peut se demander à quoi sert le premier